### PR TITLE
Deploy vue version of site

### PR DESCRIPTION
**Pull request by josh-chamberlain**
## Context
I merged #134, then did some clumsy reverting and stuff, and we are now in a state where `dev` is at the state of #134. 

## This PR
- [x] digital ocean config
  - [attempted deployment](https://cloud.digitalocean.com/apps/b31f30f3-2f77-4845-84ba-4b1d903f7caa/deployments/4f7a8aa5-6858-4197-a424-356cf5a093e0?i=feca0b)
  - brought `dev` up to date for real, and [reattempted deployment](https://cloud.digitalocean.com/apps/b31f30f3-2f77-4845-84ba-4b1d903f7caa/deployments/1ca520b6-25cd-4d79-a12b-4c7920d437fe?i=feca0b)
- [x] readme updates (convert commands to pnpm → npm)

### CODE REVIEW

**Comment by josh-chamberlain** [comment]
@joshuagraber can we work on this next week? I linked to an attempted deployment above, I think we're missing a run command but not sure what it would be. I invited you to Digital Ocean, so we should be able to poke around there. Please excuse the chaotic commit history 😬

**Comment by joshuagraber** [comment]
> @joshuagraber can we work on this next week? I linked to an attempted deployment above, I think we're missing a run command but not sure what it would be. I invited you to Digital Ocean, so we should be able to poke around there. Please excuse the chaotic commit history 😬 

@josh-chamberlain Yes, very sorry I missed your comment until now. I saw the DigitalOcean invite and will look into this tomorrow.

File: index.html
Comment (joshuagraber) [inline index.html:1]: Should have caught this in review, but this `index.html` file still had old code from pre-Vue version of site, and was also trying to import previous `nav.js` script that didn't exist. The dev server worked, but build failed. Pretty sure that's what was causing the deployment failure to DO. @josh-chamberlain  would you try again and see if that solves?

Also while here, I fixed the mobile nav stuff (hacky, but will be replaced soon by component from `design-system`. But it works for now, so IMO I think that's okay).

Also added some styles to a stylesheet in `App.vue`. Again, these will need to be removed when `design-system` fully implemented, but they're a fine stopgap for now.

Comment (josh-chamberlain) [reply index.html:1]: it's 100% possible the original submission was correct and I borked it with reversions of reverted PRs.

**Comment by mbodeantor** [comment]
@josh-chamberlain All looks good to me, except the "Data Sources" link under "Directory of Data Sources" is broken.

**Comment by josh-chamberlain** [comment]
Actually, we're seeing the same issue here as https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/81. i.e. I can click around the website fine, but if I try to navigate directly to any page (`https://pdap.dev/donate`, `/jobs`, etc) it doesn't work.

**Comment by josh-chamberlain** [comment]
Attempted to fix by updating router.js (there wasn't a variable set, but i think it was likely defaulting to `/` anyway or nothing would have worked)

Successfully fixed by reading [DO docs](https://docs.digitalocean.com/products/app-platform/how-to/manage-static-sites/) about custom pages, and added a "catchall page"

Screen Shot 2023-11-01 at 3 49 27 PM

**Comment by josh-chamberlain** [comment]
@benpolinsky FYI, this is finally done! thanks again for your help

**Comment by benpolinsky** [comment]
> @benpolinsky FYI, this is finally done! thanks again for your help

Excellent! No problem.

**Pull request merged**